src/plugins/qml/MuseScore/Plugins/internal/PluginItem.qml

```diff
@@ -50,29 +50,17 @@ Item {
         onTriggered: root.clicked()
     }
 
-    Rectangle {
-        id: thumbnailRect
+    Item {
+        id: thumbnail
 
         anchors.top: parent.top
         width: parent.width
         height: 144
 
-        color: "transparent"
-        radius: root.radius + border.width
-
-        border.color: ui.theme.fontPrimaryColor
-        border.width: root.selected ? 2 : 0
-
-        NavigationFocusBorder {
```

Comment:
Why you decided to use a custom navigation border instead of this component?

Reply:
The border for navigation and selection looks exactly the same, so I thought it is redundant to use two separate rectangles for these purposes. If you prefer, I could reuse `NavigationFocusBorder` and just override its `visible` property to make it also visible when the item is selected.

Reply:
They perform different functions and it is better not to mix them. 
It should be clear from the code where the selection rectangle is and where the navigation is.

Reply:
Okay, I separated them!